mvvm-components/src/test/kotlin/se/springworks/mvvmcomponents/fragment/BaseDataBindingFragmentTest.kt

```diff
@@ -0,0 +1,70 @@
+package se.springworks.mvvmcomponents.fragment
+
+import android.view.View
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.spy
+import com.nhaarman.mockito_kotlin.verify
+import com.vhudnitsky.mvvmcomponents.databinding.StringItemBinding
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.Before
+import org.junit.Test
+import se.springworks.mvvmcomponents.recyclerview.viewmodel.StringItemViewModel
+
+class StubBaseDataBindingFragment : BaseDataBindingFragment<StringItemBinding, StringItemViewModel>() {
+  override fun resourceBRViewModelId(): Int {
+    return 0
+  }
+
+  override fun provideViewModel(): StringItemViewModel {
+    return StringItemViewModel()
+  }
+
+  override fun getLayoutId(): Int {
+    return 0
+  }
+}
+
+class BaseDataBindingFragmentTest {
+  private lateinit var fragment: StubBaseDataBindingFragment
+
+  @Before
+  fun setUp() {
+    fragment = StubBaseDataBindingFragment()
+    fragment.onCreate(null)
```

Comment:
When running on a non-Android device this will call into the stubbed android library. I wonder if there will be any strange side effects.

Reply:
Doesn't matter for us. Because I'm testing that view models and fragment is connected well

Reply:
👍